qt/aqt/addons.py

```diff
@@ -626,7 +666,7 @@ def onDelete(self):
     def onGetAddons(self):
         GetAddons(self)
 
-    def onInstallFiles(self, paths=None):
+    def onInstallFiles(self, paths: List[str] = None, external: bool = False):
```

Comment:
If paths is optional, the type should reflect that.

Reply:
Should be fixed now!